### ISSUE TITLE
The `permitted-content` rule doesn't work on an element that includes mutable elements

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#460]
```jsx
<label>
  <p>{foo.bar}</p>
</labe>
```

## Benchmark cross-reference

_Sync'd from `tests/external/snapshots/` (submodule=1429313, nu-validator@sha256:210bbc353b23, markuplint=5.0.0-rc.4, generated=2026-08-12 06:31:53 UTC)._

**Not directly verifiable against the current nu-validator fixture suite.** Reason: `permitted-content` on elements containing mutable (preprocessor) children is a markuplint-internal concern; the nu-validator suite has no fixtures with mutable placeholders.

Fixtures could be authored manually under a separate tree, or contributed upstream to [`validator/validator`](https://github.com/validator/validator). See the [`bench-triage` skill](https://github.com/markuplint/markuplint/blob/dev/.claude/skills/bench-triage/SKILL.md) for the audit workflow.

**Comment by YusukeHirao** [comment]
There are cases where it is sufficient to check only the child elements, and there are cases where it is necessary to traverse to the descendants, and most of them should be the former, so if the children do not have a mutable node, the check can be completed.

**Comment by YusukeHirao** [comment]
The `label` element's content model: `:model(phrasing):not(label, :has(label))`.

Actually, the above sample can evaluate when using only the first condition `:model(phrasing)`. The p element is not the phrasing content. So it is invalid.

**Comment by YusukeHirao** [comment]
Verified against current `dev` HEAD — no longer reproduces:

```jsx
const X = () => (
  <label>
    <p>{foo.bar}</p>
  </label>
);
```

```
$ markuplint --config '{"parser":{"\\.jsx$":"@markuplint/jsx-parser"},"extends":["markuplint:html-standard"]}' input.jsx
error: このコンテキストでは、要素「label」に要素「p」を含めることはできません [normative] (html-standard/permitted-contents) input.jsx:3:5
```

`permitted-contents` correctly flags `<p>` (flow content) as disallowed inside `<label>` (phrasing content only), even though `<p>`'s own text child is a dynamic/mutable expression (`{foo.bar}`).

The current design (`packages/@markuplint/rules/src/permitted-contents/types.ts`, `Options.ignoreHasMutableChildren`) explains why this now works as expected: the mutable-children leniency is scoped narrowly — it only "skips reporting **missing**-node errors on elements that have mutable children" (e.g. a content model requiring at least one specific child, which a framework might inject dynamically). It does not suppress **disallowed**-content violations, and it does not propagate past the element boundary — `<p>` having dynamic text does not make `<label>`'s own content-model check treat `<p>` itself as absent/skippable. That distinction (missing-content leniency vs. disallowed-content enforcement) matches exactly what this Issue asked for.

Closing as resolved. Removed the stale `secondary` bench-xref mapping for this Issue in the same commit as #263 (both no longer applicable).
